Gx3 4620 ⚡️ tools qa elements radio buttons

DonnaCas opened the pull request:
Se confirma la terminación de las pruebas realizadas y se requiere hace merge a la rama ppal.

AlexanderWindecker commented:
Hola Donna como estas, espero te encuentres bien. Te deje unas correcciones.

Saludos.-

DonnaCas commented:
Muy buen día @AlexanderWindecker,  corregí lo que me indicaste anteriormente, serrías tan formal de revisar nuevamente, mil gracias

saiotest commented on package-lock.json:
Solicito que se ELIMINE este archivo antes de mergear, esto se debe al uso de:
`npm install` 
y la idea es usar `yarn install`
@DonnaCas @AlexanderWindecker

saiotest reviewed (changes requested):
Dejé un mensaje en el archivo package-lock.json para solicitar cambios urgentes
@DonnaCas @AlexanderWindecker

AlexanderWindecker reviewed (approved):
todo ok

saiotest committed:
chore: add comments

saiotest committed:
fix: conflict from branch GX3-4620
On branch checkForMergeQA
Changes to be committed:
	new file:   .github/workflows/sanityDonnaCas.yml
	new file:   coverage/S45/GX3-4620.md
	new file:   cypress/e2e/Tests/Elements/GX3-4620-Radio-Buttons.cy.js
	modified:   yarn.lock
